sky/shell/platform/ios/framework/Source/accessibility_bridge.h

```diff
@@ -24,6 +24,36 @@
 
 namespace sky {
 namespace shell {
+class AccessibilityBridge;
+}
+}
+
+@interface AccessibilityNode : NSObject
+
+/*
```

Comment:
Comments are `headerdoc` style (just like `javadoc`). So something like

```
/**
 *  Blah
 */
```

If you are using Xcode, https://github.com/onevcat/VVDocumenter-Xcode is handy in generating these.